pkg/auditor/auditor.go

```diff
@@ -245,3 +246,19 @@ func (a *Auditor) unmarshalRegistryV0(b []byte) (map[string]*RegistryEntry, erro
 	}
 	return registry, nil
 }
+
+func (a *Auditor) GetLastCommitedOffsetV0(identifier string) (int64, int) {
```

Comment:
I feel like this really belongs to `unmarshalRegistryV0`. We should hydrade the v0 registry into a v1 registry with the `file:` prefix.

Reply:
way easier indeed